src/dllmain.cpp

```diff
@@ -0,0 +1,20 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+#define NOMINMAX
+#include <Windows.h>
+
+BOOL APIENTRY DllMain( HMODULE hModule,
```

Comment:
according to docs `DllMain` is an optional entry point: https://docs.microsoft.com/en-us/windows/win32/dlls/dllmain

None the less no cpp files

Reply:
Yes, this wasn't supposed to be included, my mistake.